RELEASE.md

```diff
@@ -0,0 +1,63 @@
+# Release process for a new version of the CF conventions
+Complete the following steps in the order presented.
+
+## Documents
+
+* Check that the version (e.g. `1.10`) in `cf-conventions.adoc` is correct.
+* Set the release date in `cf-conventions.adoc`.
+* Check that the version in `conformance.adoc` is correct.
+* Check that the history in `history.adoc` is up to date. Every Pull
+  Request merged to the convention document should be
+  referenced. Check that all merged Pull Requests that contributed to
+  the new version have the appropriate GitHub Milestone attached to
+  them.
+
+
+## Repository
+When all Pull Requests have been merged:
+
+* Tag the `main` branch with the new version (e.g. `v1.10.0`).
+* Create a new "Latest release" that contains a link to the Pull
+  Requests that contributed to the new version. Include a link to the
```

Comment:
Sentence starting on this line looks like it might be a leftover from something.

What is the appropriate GitHub Milestone?